sbncode/EventGenerator/MeVPrtl/config/numi_kaon_common.fcl

```diff
@@ -11,6 +11,10 @@ icarus_numi_beam2detector_rotation: [   0.921228671,    0.0226872648,   0.388359
 
 icarus_numi_beam_origin: [0, 0, 0, 4.8195e2, 79.5800e2, 794.3955e2] # cm
 
+# icarus_detector_box: [-364.49, 364.49, -173.41, 143.41, -909.951, 879.951]
+# icarus_detector_box: [-369.63. 369.63, -224.61, 177.71, -994.950652270838, 994.950652270838]
+icarus_detector_box: [-378.49, 378.49, -191.86, 144.96, -904.950652270838, 904.950652270838]
```

Comment:
Now this is something that `Geometry` knows.
I _recommend_ to try to figure out how to get this from LArSoft for the future.
@grayputnam: is this anything that has already been discussed? if not, please open an issue about it.

P.S. I really appreciate the centralisation of these coordinates into a single point.